src/couch_changes.erl

```diff
@@ -339,15 +348,8 @@ check_docids(_) ->
 
 
 open_ddoc(#db{name=DbName, id_tree=undefined}, DDocId) ->
-    {_, Ref} = spawn_monitor(fun() ->
-        exit(fabric:open_doc(mem3:dbname(DbName), DDocId, [ejson_body]))
-    end),
-    receive
-        {'DOWN', Ref, _, _, {ok, _}=Response} ->
-            Response;
-        {'DOWN', Ref, _, _, Response} ->
-            throw(Response)
-    end;
+    {ok, DDoc} = ddoc_cache:open_doc(mem3:dbname(DbName), DDocId),
```

Comment:
Sure, I usually like to be explicit about it, but switching to `{ok, _DDoc}` is fine with me.